src/config/settings.py

```diff
@@ -227,3 +227,11 @@
 DEFAULT_FILE_STORAGE = os.getenv(
     "DEFAULT_FILE_STORAGE", "storages.backends.s3boto3.S3Boto3Storage"
 )
+
+# SECURITY
+# https://docs.djangoproject.com/en/4.0/ref/settings/#csrf-trusted-origins
+allowed_csrf_origins = os.getenv("CSRF_TRUSTED_ORIGINS", "http://127.0.0.1")
```

Comment:
It seems that Django 4 introduced new checks to this value: https://docs.djangoproject.com/en/4.0/releases/4.0/#csrf-trusted-origins-changes-4-0

So `CSRF_TRUSTED_ORIGINS` cannot be empty. Either it is set with the correct constraints or not set at all.

I would probably wrap the setting of `CSRF_TRUSTED_ORIGINS` under the condition that it has a non-null/non-blank value.

ie.: I'd keep the Django defaults as much as possible.

Reply:
Which is funny, because the default is in fact an empty list https://docs.djangoproject.com/en/4.0/ref/settings/#csrf-trusted-origins

Reply:
True, which with your changes should've been fine since `CSRF_TRUSTED_ORIGINS` would just be empty. But it seems that there's a check if it is set by the application and if It is then it cannot be empty which is strange.

```
?: (4_0.E001) As of Django 4.0, the values in the CSRF_TRUSTED_ORIGINS setting must start with a scheme (usually http:// or https://) but found . See the release notes for details.
```